PL_2_komentarze.py

```diff
@@ -0,0 +1,2722 @@
+    global wsX
+    wsX = wbX.sheet_by_index(0)
+    listaklientow.clear()
+    for row in wsX.col(0):
+        key = b'Uf-FryHfa-JXDToRDClFsSrv2h56kzEUCK5imvQngbo='
+        f = Fernet(key)
+        row_value = row.value
+        kodowanie = row_value.encode()
+        odkryte = f.decrypt(kodowanie)
+        odszyfrowane = odkryte.decode('utf-8')
+        listaklientow.append(odszyfrowane)
+
+ladujklientow()
+
+def dodajklienta():
+    try:
+        wbnX = xlrd.open_workbook(
+            '//Srv-dokumenty/Swiadectwomat/bin/klient.xls',
+            formatting_info=True)
+        adresklienta = '//Srv-dokumenty/Swiadectwomat/bin/klient.xls'
+    except FileNotFoundError:
+        messagebox.showerror('Zapis lokalny niemożliwy!',
+                               'Dodanie klienta do bazy nie jest możliwe ze względu na brak połączenia ze spisem na serwerze sieciowym!')
+        pass
+    wsnX = wbnX.sheet_by_index(0)
+    wbn1X = copy(wbnX)
+    wsn1X = wbn1X.get_sheet(0)
+    liczbawierszy = 0
+    for _ in wsnX.col(0):
+        liczbawierszy += 1
+    dk1 = entryKlient1.get()
+    dk2 = entryKlient2.get()
+    dk3 = entryKlient3.get()
+    key = b'Uf-FryHfa-JXDToRDClFsSrv2h56kzEUCK5imvQngbo='
+    f = Fernet(key)
+    dk1token = f.encrypt(bytes(dk1, 'utf-8'))
+    dk2token = f.encrypt(bytes(dk2, 'utf-8'))
+    dk3token = f.encrypt(bytes(dk3, 'utf-8'))
+    dk11token = dk1token.decode('utf-8')
+    dk22token = dk2token.decode('utf-8')
+    dk33token = dk3token.decode('utf-8')
+    daneklienta = [dk11token, dk22token, dk33token]
+    potwierdzeniedodaniaklienta = messagebox.askyesno('Dodanie nowego klienta', 'Czy na pewno chcesz dodać następującego klienta:\n\n'+dk1+'\n'+dk2+'\n'+dk3)
+    if potwierdzeniedodaniaklienta:
+        for col_index in range(0,3):
+            wsn1X.write(liczbawierszy, col_index, daneklienta[col_index])
+        wbn1X.save(adresklienta)
+        messagebox.showinfo("Dodano klienta do listy!", "Dodano klienta do listy!\n\nGratuluję sukcesu!")
+        ladujklientow()
+    else:
+        pass
+
+tree_columns = ("Nazwa klienta", "Adres 1", "Adres 2")
+def sortby(tree, col, descending):
+    """Sort tree contents when a column is clicked on."""
+    # grab values to sort
+    data = [(tree.set(child, col), child) for child in tree.get_children('')]
+
+    # reorder data
+    data.sort(reverse=descending)
+    for indx, item in enumerate(data):
+        tree.move(item[1], '', indx)
+
+    # switch the heading so that it will sort in the opposite direction
+    tree.heading(col,
+        command=lambda col=col: sortby(tree, col, int(not descending)))
+
+class Drzewo(object):
+    def __init__(self, rodzic, tree_data, entry1, entry2, entry3):
+        self.tree = None
+        self._setup_widgets(rodzic)
+        self._build_tree(tree_data)
+        self.tree.bind('<Double-1>', self.daneklienta)
+
+    def _setup_widgets(self, rodzic):
+        container = ttk.Frame(rodzic)
+#         container.pack(fill='both', expand=True)
```

Comment:
Warto korzystać z gita, wtedy każdy zakomentowany/zbędny kod bez skrupułów wypierdalasz. A jeśli się kiedyś okaże że jednak może się przydać, to zaglądasz w poprzednie wersje pliku na gicie :)